doc/content/verification_and_validation/ver-1fc.md

```diff
@@ -26,36 +29,27 @@ where
 - $L_i$ is Length of segment $i$ ($L_A=L_B=40$ cm),
 - $k_i$ is thermal conductivity of segment $i$ ($k_A = 401$ W/m/K, $k_B = 80.2$ W/m/K).
 
-At steady state, the flux in and out of any section of the slab are equal.
-The temperature at the interface ($T_I$) can be found by setting the flux through A equal to the flux
-through B, which leads to:
+At steady state, the flux in and out of any section of the slab are equal. The temperature at the interface ($T_I$) can be found by setting the flux through A equal to the flux through B, which leads to:
 
 \begin{equation} \label{eq:solution_analytical_steady_state}
 \frac{T_{SA} - T_{I}}{\left(L_A / k_A \right)} = \frac{T_{I} - T_{SB}}{\left(L_B / k_B \right)}.
 \end{equation}
 
-The interface temperature at steady state is therefore equal to $T_I = 500$ K. The temperature profile
-for conduction in steady state, with constant physical properties, is linear. The temperature
-profile of A and B can therefore be found through linear interpolation.
+The interface temperature at steady state is therefore equal to $T_I = 500$ K. The temperature profile for conduction in steady state, with constant physical properties, is linear. The temperature profile of A and B can therefore be found through linear interpolation.
 
-With TMAP8, the steady state solution can be obtained in different ways: by using the [steady state solve](source/executioners/Steady.md) or by running a [transient simulation](source/executioners/Transient.md) until steady state is reached.
-[!cite](ambrosek2008verification) indicates that the steady state solution was obtained by running the transient solution until $t=10,000$ s, which is what is reproduced with TMAP8 here.
-TMAP8 predictions were found to be identical to the analytical solution with a root mean square percentage error (RMSPE) of 0.23 %, as shown in
-[ver-1fc_comparison_temperature_steady_state].
+With TMAP8, the steady state solution can be obtained in different ways: by using the [steady state solve](source/executioners/Steady.md) or by running a [transient simulation](source/executioners/Transient.md) until steady state is reached. [!cite](ambrosek2008verification) indicates that the steady state solution was obtained by running the transient solution until $t=10,000$ s, which is what is reproduced with TMAP8 here. TMAP8 predictions were found to be identical to the analytical solution with a root mean square percentage error (RMSPE) of 0.23 %, as shown in [ver-1fc_comparison_temperature_steady_state].
```

Comment:
```suggestion
With TMAP8, the steady state solution can be obtained in different ways: by using the [steady state solve](source/executioners/Steady.md) or by running a [transient simulation](source/executioners/Transient.md) until steady state is reached. 
[!cite](ambrosek2008verification) indicates that the steady state solution was obtained by running the transient solution until $t=10,000$ s, which is what is reproduced with TMAP8 here. 
TMAP8 predictions were found to be identical to the analytical solution with a root mean square percentage error (RMSPE) of 0.23 %, as shown in [ver-1fc_comparison_temperature_steady_state].
```
Although I prefer not to create a new line in the middle of a sentence, it sometimes help to do it after a sentence - even in the middle of a paragraph. It makes reviews easier.